packages/data/src/lookitS3.ts

```diff
@@ -1,32 +1,43 @@
-import { S3 } from "@aws-sdk/client-s3";
+import {
+  CompleteMultipartUploadCommand,
+  CreateMultipartUploadCommand,
+  S3Client,
+  UploadPartCommand,
+} from "@aws-sdk/client-s3";
 import { Env } from "./types";
 
 /** Provides functionality to upload videos incremetally to an AWS S3 Bucket. */
-export default class {
+class LookitS3 {
   private blobParts: Blob[];
   private promises: Promise<{ PartNumber: number; ETag?: string }>[];
   private partNumber: number;
   private partsUploaded: number;
-  private s3: S3;
+  private s3: S3Client;
   private uploadId?: string;
   private env: Env;
   private key: string;
 
   /**
-   * Provide file name and AWS secrets to upload file to a S3 bucket.
+   * Provide file name to initiate a new upload to a S3 bucket. The AWS secrets
+   * and bucket name come from environment variables.
    *
    * @param key - Used to identify upload, mostly likely video file name.
-   * @param s3vars - Object with secrets and bucket name.
    */
-  public constructor(key: string, s3vars: Env) {
-    this.env = s3vars;
+  public constructor(key: string) {
+    this.env = {
+      accessKeyId: process.env.S3_ACCESS_KEY_ID,
+      bucket: process.env.S3_BUCKET,
+      secretAccessKey: process.env.S3_SECRET_ACCESS_KEY,
+      region: process.env.S3_REGION,
+    } as Env;
```

Comment:
The only value from `Env` that is need throughout the class is the bucket value.  We can probably get rid of the `Env` type and Instantiate the S3 Client with values from the environment.